sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.aest.controller.assessOfferingChangeRequest.e2e-spec.ts

```diff
@@ -305,6 +308,71 @@ describe("EducationProgramOfferingAESTController(e2e)-assessOfferingChangeReques
     },
   );
 
+  it(
+    "Should determine in-progress applications for a requested offering when the offering change is approved" +
+      "and there are applications associated with the requested offering.",
+    async () => {
```

Comment:
I couldn't follow the test description here.

The test is validating 
the offering change for an offering with one or more applications not in completed status being approved.